companion/src/simulation/widgets/buttonswidget.h

```diff
@@ -75,30 +75,30 @@ class ButtonsWidget : public QWidget
         int key;
         const char * bitmap;
     };
-      
+
     void setBitmap(const char * bitmap)
     {
       if (bitmap) {
         QWidget::setStyleSheet(QString("background:url(:/images/simulator/%1);").arg(bitmap));
       }
     }
-    
+
     virtual void mousePressEvent(QMouseEvent * event)
     {
       int x = event->x();
       int y = event->y();
       setFocus();
       if (event->button() == Qt::LeftButton) {
-        foreach(Area area, areas) {
-          if (area.contains(x, y)) {
+      foreach(Area area, areas) {
```

Comment:
This was the old version... I think we're looking at an individual commit here? I did the file move in one commit and then the actual changes later on (my preferred method when possible) 

LOL, man, that confused the heck out of me for about 2 minutes!  😆